src/atoms/Icon/index.js

```diff
@@ -13,7 +14,7 @@ function Icon ( props ) {
 
   return (
     <div
-      className={ classnames(className, wrapperClassName) }
+      className={ classnames(styles['icon-wrapper'], className, wrapperClassName) }
```

Comment:
adding `icon-wrapper` class to this looks like it makes the icons disappear in the styleguide examples